### ISSUE TITLE
Localization/Translation of Imap Folders

### ISSUE THREAD

**Issue by benjamindahl** [nextcloud/mail#2725]
### Feature Request

Using the Mail App in nextcloud shows me the English Imap folder names. Would be great to have them in German like in other mail apps (Thunderbird, SOGo, Outlook, etc.)
Inbox = Posteingang
Sent = Gesendet
Trash = Papierkorb
Drafts = Entwürfe
etc.

Cheers
Benny

**Comment by ChristophWurst** [comment]
Yep, we actually did that and it broke #2290. The next release will have localized mailboxes again :)

**Comment by ChristophWurst** [comment]
Patch was at https://github.com/nextcloud/mail/pull/2645
